.github/pull_request_template.md

```diff
@@ -1,37 +1,48 @@
-Fixes | Closes | Resolves #
+# Description
 
-> Anything marked as optional that you didn't need to fill in, please remove it from the PR description. Choose one of the keywords above to refer to the issue this PR solves, followed by the issue number (e.g Fixes # 666). If there is no issue, remove the line. Remove this note after reading.
+>Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change. 
+>**THIS WILL BE USED IN RELEASE NOTES**
 
-## Changes:
--
+Description here
 
-## Types of changes
+Fixes # (issue)
 
-What types of changes does your code introduce?
-_Put an `x` in the boxes that apply_
+## Type of change
 
-- [ ] Bugfix (non-breaking change which fixes an issue)
+Please delete options that are not relevant.

```

Comment:
If it asks to delete irrelevant items, then what's the point of having checkboxes here? One will just delete the irrelevant ones or in the case of checkboxes, will check only the relevant ones as we do now.

Reply:
@rubo Will remove this line :)